OntologyMapper/src/OntologyMapper/OntologyMappingManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="OntologyMappingManager.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.SmartFacilities.OntologyMapper
+{
+    using Microsoft.Azure.DigitalTwins.Parser;
+    using Microsoft.Azure.DigitalTwins.Parser.Models;
+
+    public class OntologyMappingManager : IOntologyMappingManager
+    {
+        private OntologyMapping ontologyMapping;
+
+        public OntologyMappingManager(IOntologyMappingLoader mappingLoader)
+        {
+            ontologyMapping = mappingLoader.LoadOntologyMapping();
+        }
+
+        public bool TryGetInterfaceRemapDtmi(Dtmi inputDtmi, out DtmiRemap? dtmiRemap)
+        {
+            dtmiRemap = ontologyMapping.InterfaceRemaps.FirstOrDefault(r => r.InputDtmi == inputDtmi.ToString() && !r.IsIgnored);
+
+            if (dtmiRemap != null)
```

Comment:
Since its FirstOrDefault wouldn't the value not be null?

Reply:
Default is null